README.md

```diff
@@ -1,135 +1,131 @@
+The version represents which output bundle version is currently being used. The current version is v1.
 
-## Deploy your app to Firebase Hosting
+### RunConfig
 
-When you're ready to share your changes with the world, deploy your app to your live site:
+The runConfig fields are in charge of configuring the backend’s Cloud run deploy.
 
-1. Run `firebase deploy` from the terminal. This will build your application, determine if a backend is needed, and if so build and deploy a Cloud Function for you.
-3. Check your website on: `SITE_ID.web.app` or `PROJECT_ID.web.app` (or your custom domain, if you set one up)
-
-## Configuring your backend
+```typescript
+interface RunConfig {
+  runCommand: string;
+  environmentVariables: EnvVarConfig[];
+  concurrency?: number;
+  cpu?: number;
+  memoryMiB?: number;
+  minInstances?: number;
+  maxInstances?: number;
+}
 
-In your `firebase.json` you can alter the configuration of the code-generated Cloud Function by editing the "frameworksBackend"
-option. "frameworksBackend" takes the same options as [firebase-functions/v2/https.httpsOptions](https://firebase.google.com/docs/reference/functions/2nd-gen/node/firebase-functions.https.httpsoptions)
-though JSON-serializable. E.g,
+interface EnvVarConfig {
+  variable: string;
+  value: string;
+  availability: 'RUNTIME'
+}
 
+```
 
-```json
-{
-  "hosting": {
-    "source": ".",
-    "ignore": [
-      "firebase.json",
-      "**/.*",
-      "**/node_modules/**"
-    ],
-    "frameworksBackend": {
-      "region": "us-central1",
-      "minInstances": 1,
-      "maxInstances": 10
-    }
-  }
+| Field  | Type | Description |
+| ---------- | ------- | - |
+| runCommand | string |Command to start the server (e.g. "node dist/index.js"). Assume this command is run from the root dir of the workspace.This should be the productionized version of the server start command. |
+| environmentVariables| EnvVarConfig[] | Environment variables present in the server execution environment.|
+| concurrency | number | The maximum number of concurrent requests that each server instance can receive.|
+| cpu | number |The number of CPUs used in a single server instance.. |
+| memoryMiB | number | The amount of memory available for a server instance.|
+| minInstance | number |The limit on the minimum number of function instances that may coexist at a given time. |
+| MaxInstance | number | The limit on the maximum number of function instances that may coexist at a given time.|
+| EnvVarConfig.variable | string |Name of the environment variable |
+| EnvVarConfig.value | string |Value associated with the environment variable |
+| EnvVarConfig.availability | RUNTIME | Where the variable will be available, for now this will always be RUNTIME |
+
+Many of these fields are shared with apphosting.yaml as well so see [our documentation](https://firebase.google.com/docs/reference/apphosting/rest/v1beta/projects.locations.backends.builds#runconfig) for additional context.
```

Comment:
I'd suggest:

"Many of these fields are shared with apphosting.yaml. See the [runConfig reference documentation](https://firebase.google.com/docs/reference/apphosting/rest/v1beta/projects.locations.backends.builds#runconfig) for additional context."

Reply:
Am I seeing an outdated diff?  It looks like you did not include the suggested change (as well as various other nits)